test/functional/tests/io_class/test_io_class_lvm_on_cas_dss.py

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache import Cache
+from api.cas.cache_config import CacheMode
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from storage_devices.lvm import Lvm, LvmConfiguration
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite
+from test_utils.size import Size, Unit
+
+mountpoint = "/mnt/"
+io_target = "/mnt/test"
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_io_class_lvm_on_cas_dss():
+    """
+        title: IO class for CAS device behind LVM.
+        description: Validate the ability of CAS to cache IO class when CAS device is used as LVM.
+        pass_criteria:
+          - Create CAS device and LVM on top of it successfully.
+          - Loading IO class configuration successfully.
+          - Running FIO for file size from IO class from 11 to 21 successfully.
+          - Increasing proper statistics as expected.
+    """
+    with TestRun.step(f"Create CAS device."):
+
+        cache_dev, core_dev = prepare_devices()
```

Comment:
`prepare_devices()` returns lists, but the names of these variables are singular, not plural. Please change it to plural or return the first elements of both lists, you don't use more anyway. Or even close `cache` and `core` creation inside `prepare_devices()` and return them. IMO it would be more readable.